l10n/es.js

```diff
@@ -0,0 +1,17 @@
+OC.L10N.register(
+    "facerecognition",
+    {
+    "Persons" : "Personas",
+    "Looking for faces in this image..." : "Buscando por rostros en esta imagen...",
```

Comment:
... is usually ellipsis sign these days (…)

Reply:
I still resist to unicode...  :sweat_smile: